### PR TITLE
Add migration for trix, convert fields from markdown to html.

### LINKED ISSUE #1527: Migration bestehende Markdown-Protokolle
Die bestehenden Protokolle im Markdown-Format müssen teilweise automatisch nach HTML migriert werden.

Mindestens Formatierungen und Listenaufzählungen sollten übernommen werden.

### PR DESCRIPTION
**Pull request by deiferni**
This migration converts fields from markdown to html for each `AgendaItem` and `Proposal`. This is required since we switched the editor to [trix](https://github.com/basecamp/trix), see #1487 and #1524. The migration is done independently for each table since the relation between agenda_items and proposals is weak and optional (there may be agenda_items without a proposal and vice versa).

The conversion is done with [Markdown](https://pypi.python.org/pypi/Markdown/2.0.3), the package seems to leave html alone when re-run with html input, so no additional checks are required.

I have tested the migration with data copied from a meeting on a productive system.

Closes #1527.

### CODE REVIEW

**Comment by deiferni** [comment]
@lukasgraf plz have a look

File: opengever/meeting/upgrades/to4630.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+from markdown import Markdown
```
Comment (lukasgraf) [inline opengever/meeting/upgrades/to4630.py]: Doesn't this require a dependency in `setup.py`?

**Comment by lukasgraf** [comment]
Other than the missing dependency: Looks good to me! :+1:

**Comment by lukasgraf** [comment]
Rebased on behalf of @deiferni

**Comment by deiferni** [comment]
@lukasgraf updated, `setup.py` dependency added

**Comment by lukasgraf** [comment]
:+1:

**Pull request merged**